tests/android-diff-size.test.ts

```diff
@@ -0,0 +1,135 @@
+jest.mock("danger", () => jest.fn());
+import danger from "danger";
+const dm = danger as any;
+
+import diffSize from "../org/pr/android-diff-size";
+
+// The mocked data and return values for calls the rule makes.
+beforeEach(() => {
+    dm.warn = jest.fn().mockReturnValue(true);
+
+    dm.danger = {
+        git: {
+            modified_files: [],
+            created_files: [],
+            deleted_files: []
+        },
+        github: {
+            pr: {
+                additions: 200,
+                deletions: 10  
+            },
+            issue: {
+                labels: []
+            },
+        },
+    };
+});
+
+describe("PR diff size checks", () => {
+    it("does not warn with less than 300 lins", async () => {
+        await diffSize();
+        
+        expect(dm.warn).not.toHaveBeenCalled();
+    })
+
+    it("warns with more than 300 lines", async () => {
+        dm.danger.github.pr.deletions = 101;
```

Comment:
Completely opinionated so feel free to ignore, but I wonder if it wouldn't be easier to read if the mocked data were 0,0 and you did set both `pr.additions` and `pr.deletions`'s values in each test?

Because I almost commented saying "how does this test pass given your limit is 300 but you only use 101 which is less than 300" and it took me a second to realize that `pr.additions` had a default value of 200 in the mocks and that's where it all came from.

Reply:
Oh, I see! I find it slightly easier to have a default and have only the diff for that particular test in the test code, but I'm totally fine with both. 
I think that most of tests in this repository are organized this way though, so probably we should keep it for consistency.